File: php/item/getPricingTiers.php
<?php

use ultracart\v2\ApiException;

require_once '../vendor/autoload.php';
require_once '../samples.php';

$item_api = Samples::getItemApi();


try {
    /*
     * Possible expansion values for PricingTier object:
     * approval_notification
     * signup_notification
     */

    $expand = "approval_notification,signup_notification";
    $api_response = $item_api->getPricingTiers($expand);

} catch (ApiException $e) {
    echo 'ApiException occurred.';
    var_dump($e);
    die(1);
}

var_dump($api_response->getPricingTiers());

File: python/item/get_pricing_tiers.py
from ultracart.apis import ItemApi
from samples import api_client

try:
    """
    Possible expansion values for PricingTier object:
    - approval_notification
    - signup_notification
    """
    item_api = ItemApi(api_client())

    expand = "approval_notification,signup_notification"
    api_response = item_api.get_pricing_tiers(expand=expand)

except Exception as e:
    print('Exception occurred.')
    print(e)
    raise

print(api_response.get_pricing_tiers())